schema.graphql

```diff
@@ -0,0 +1,32 @@
+type User (
+  id: Int!
+  bio String
+  chirps: [Chirp]
+  created_at DateTime!
+  follows: [Follow]
+  likes: [Like]
+  password String!
+  pinnedChirp Int
+  privacyStatus Int!
+  profilePicture String
+  username: String!
+)
+
+type Chirp(
+  id: Int!
+  content String!
+  created_at DateTime!
+  user_id Int!
+)
+
+type Like(
+  user_id User.id!
+  chirp_id Chirp.id!
+  created_at DateTime!
+)
+
+type Follow(
```

Comment:
Retweets are something that I'm still wrapping my head around. Maybe an interface for a tweet and then implement a retweet with other fields

Reply:
That makes sense, because we'd need to have the IDs of the author and the Rechirper - My wonder is how would that work in the flow of querying for all Chirps from the user? Do you think it would actually be good to have Rechirps implemented after Chirps are done?

Reply:
That sounds good as long as we have a plan for implementing them. I'd rather not have to change too much to support it